dist/scss/wds-components/_global-navigation-search.scss

```diff
@@ -234,6 +234,59 @@ $toggle-active-margin-right: 17px;
 	}
 }
 
+.wds-is-wikia-org {
```

Comment:
Can we move all of wikia.org related styles to separate files and load them conditionally only on wikia.org communities?

Reply:
https://wikia-inc.atlassian.net/browse/IW-1904